ui/src/containers/SampleGridTableContainer.jsx

```diff
@@ -976,7 +983,8 @@ export default function SampleGridTableContainer(props) {
         />
       );
     }
-    if (isSingleCell || isMOCK) {
+
+    if (isSingleCell || isMOCK || (isFLEX && isSingleCell)) {
```

Comment:
I don't think `&& isSingleCell` does anything: :thinking:

- if it's `true` then `isSingleCell ||` at the start already passes the condition;
- if it's `false` then it doesn't matter what `isFLEX` is; it never passes the condition (i.e. so adding `|| (isFLEX && isSingleCell)` doesn't change the behaviour compared to what was there before).

Reply:
correct, the extra "|| (isFLEX && isSingleCell)" is redundant. thanks.
i forgot that i added the  && !isSingleCel in the first condition.